src/App.js

```diff
@@ -13,6 +13,9 @@ ReactGA.pageview(window.location.pathname + window.location.search);
 
 const queryString = require('query-string');
 
+const deviceIsMobile = isMobile;        // HACK to allow easy mocking of isMobile for testing/debugging
+// const deviceIsMobile = true;        // uncomment this line to force app to render as if on a mobile device
```

Comment:
I’d remove this line before merge

Reply:
Why remove it?  It provides a simple way to coerce the UI to act like a mobile device - making debugging mobile simple.  If there is another way to force react-device-detect to return isMobile==true, then I would be happy to remove this code, but I looked online and did not see a way to tell react-device-detect to act like the device isMobile.

Reply:
It’s not a big deal obviously, but a couple reasons

1) best way to do mobile testing of a website imo is to use the mobile device simulator built into your browser of choice https://developers.google.com/web/tools/chrome-devtools/device-mode/

If you use that, you won’t need to mess with commenting and uncommenting parts of code to check on mobile behavior.

2) I don’t personally like to leave commented out code sitting around getting into production. You actually can’t just uncomment that line, you’d need to comment out the line above. I think a simple comment “set to true to force mobile mode for debugging” would be as helpful guidance for future debugging with less noise on the page. If this is something someone would need to do *a lot* then I can see leaving it in like this, but if it’s a lot of commenting/uncommenting, then that strikes me as an anti-pattern, and I would also ask the question, “is there a better way?” In this case, I think there is!

Reply:
Thanks for asking! Hope that is helpful!! What is merged in is also _fine_